We.Sell.Bread.Infrastructure/DataFiles/Product.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "ProductId": "411fbad3-925e-4044-9269-f962641f9277",
+    "ProductName": "Pumpernickel Bread",
+    "Price": 20.00,
+    "Description": "A dense, dark bread made from coarsely ground whole rye grains",
+    "StockQuantity": 7
+  },
+  {
+    "ProductName": "Multigrain Bread",
```

Comment:
this product has no id, either delete it from the file or add id

Reply:
I have added the missing productId